app/services/organizations/safe_remote_profile_image_url.rb

```diff
@@ -0,0 +1,13 @@
+module Organizations
+  module SafeRemoteProfileImageUrl
+    # Basic check for nil and blank URLs, alongside likely incomplete URLs, such as just "image.jpg".
+    def self.call(url)
+      if url&.start_with?("http")
```

Comment:
Cool approach to include this module, I dig it!

This check seems like it could be doing more work though, as long as we're bothering.

What if we fed the URL to `FastImage.type` (FastImage is a [library we already have in the project](https://github.com/sdsykes/fastimage)) which could more thoroughly verify that this is indeed an appropriate input.

Does that make sense?

Reply:
Neat, so rather than just setting https and firing away for whatever's there, we can take a peek at the URI first for more info about the resource before deciding it's valid. Can do!

Reply:
@dukegreene I went ahead and approved this because this isn't entirely necessary, just a thought.

But you're good to go with merging from my perspective. ✅

Reply:
When I try to add FastImage.type to the check here, I end up getting a bunch of "can't run HTTP with this spec" errors. About 70 in total. I think one way to approach it might be to add to the request stub we have...

```
stub_request(:any, /dummyimage.com/)
      .to_return(status: 200, body: "", headers: {})
```
... so that the headers we mock up allow FastImage to move ahead with the steps in `safe_remote_profile_image_url`.

It might be its own issue worth of work, if there are issues with changes to the stub breaking existing specs.